src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -143,6 +143,8 @@ def load_command_table(self, _):
         g.custom_command('bind', 'bind_ssl_cert', exception_handler=ex_handler_factory(), validator=validate_app_or_slot_exists_in_rg)
         g.custom_command('unbind', 'unbind_ssl_cert', validator=validate_app_or_slot_exists_in_rg)
         g.custom_command('delete', 'delete_ssl_cert', exception_handler=ex_handler_factory())
+        g.custom_command('import', 'import_ssl_cert', exception_handler=ex_handler_factory(), is_preview=True)
+        g.custom_command('create', 'create_ssl_cert', exception_handler=ex_handler_factory(), is_preview=True)
```

Comment:
Give we have AppService Certificates & Managed Certificates  this is targetting Free/Mamanged Certs only we should update this to name to reflect that this is for managed certificate by saying create-managed-certificate or something like that?

Reply:
Managed Certificate removed for now.